app/controllers/home_controller.rb

```diff
@@ -1,7 +1,13 @@
 class HomeController < ApplicationController
 
   def index
-    @links = Link.page(page).per(20)
+    case index_params[:sort]
+    when 'new'
+      @links = Link.order(created_at: :desc).page(page).per(20)
+    else
+      @links = Link.order(votes_count: :desc).page(page).per(20)
+    end
```

Comment:
Yeah, wrote it this way so it would be easy to add different sort methods in the future. But since I won't be doing any of those later iterations this will work great too. Thanks for the comment!